### PR TITLE
Add lustre slurm blueprint for v6 version and integration test

### PR DESCRIPTION
**Pull request by harshthakkar01**
### Submission Checklist

Please take the following actions before submitting this pull request.

* Fork your PR branch from the Toolkit "develop" branch (not main)
* Test all changes with pre-commit in a local branch [#](https://goo.gle/hpc-toolkit#development)
* Confirm that "make tests" passes all tests
* Add or modify unit tests to cover code changes
* Ensure that unit test coverage remains above 80%
* Update all applicable documentation
* Follow Cloud HPC Toolkit Contribution guidelines [#](https://goo.gle/hpc-toolkit-contributing)

### CODE REVIEW

File: tools/cloud-build/daily-tests/blueprints/lustre-slurm-v6.yaml [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+    use:
+    - network
+    # - ubuntu_partition
+    - rocky_partition
+    - centos_partition
+    - lustre
+    - slurm_login
+    settings:
+      disable_controller_public_ips: false
+      # cloud_parameters:
```
Comment (cboneti) [inline tools/cloud-build/daily-tests/blueprints/lustre-slurm-v6.yaml:144]: Perhaps remove all the commented lines? Why are they here?

File: tools/cloud-build/daily-tests/builds/lustre-slurm-v6.yaml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+---
+tags:
+- m.DDN-EXAScaler
+- m.pre-existing-vpc
+- m.schedmd-slurm-gcp-v6-controller
+- m.schedmd-slurm-gcp-v6-login
+- m.schedmd-slurm-gcp-v6-nodeset
+- m.schedmd-slurm-gcp-v6-partition
+- slurm6
```
Comment (cboneti) [inline tools/cloud-build/daily-tests/builds/lustre-slurm-v6.yaml:23]: Do we have tags for OS versions, perhaps we should, in this case, it is testing Rocky 8.

**Review by cboneti — COMMENTED** [review]
I left a few comments.

File: tools/cloud-build/daily-tests/builds/lustre-slurm-v6.yaml (hunk 2 above)
Comment (harshthakkar01) [reply tools/cloud-build/daily-tests/builds/lustre-slurm-v6.yaml:23]: I dont think so if tags are OS specific. (I didnt see anyone so far). 
and this bp is currently testing centos and rocky 8.

File: tools/cloud-build/daily-tests/blueprints/lustre-slurm-v6.yaml (hunk 1 above)
Comment (harshthakkar01) [reply tools/cloud-build/daily-tests/blueprints/lustre-slurm-v6.yaml]: There are coming from slurm-gcp v5 version. I removed these commended parameters for controller. For ubuntu compute nodes, it seems in v5 we had 3 os and removed ubuntu for some reason.

Comment (harshthakkar01) [reply tools/cloud-build/daily-tests/blueprints/lustre-slurm-v6.yaml]: lmkwyt.

**Review by cboneti — APPROVED** [review]

**Pull request merged**